metron-interface/metron-alerts/src/app/alerts/alert-details/alert-details.component.spec.ts

```diff
@@ -104,7 +104,9 @@ describe('AlertDetailsComponent', () => {
         AuthenticationService,
         AlertsService,
         UpdateService,
-        GlobalConfigService,
+        { provide: GlobalConfigService, useValue: {
```

Comment:
This line was causing an error in the console as unit tests ran because the service never returned an observable to subscribe to. It probably went unnoticed because the tests themselves wouldn't fail. Adding this stub removes the error.